docs/src/components/landing/DemoVideo.js

```diff
@@ -16,35 +16,48 @@ const Video = styled('video')(({ theme }) => ({
 export default function DemoVideo() {
   return (
     <Box
-      sx={(theme) => ({
-        borderStyle: 'solid',
-        maxWidth: 1100,
-        height: { xs: 200, sm: 390, md: 445, lg: 615 },
-        width: {
-          xs: 360,
-          sm: theme.breakpoints.values.sm + 100,
-          md: theme.breakpoints.values.md - 100,
-          lg: theme.breakpoints.values.lg - 100,
-        },
-        borderColor:
-          theme.palette.mode === 'dark' ? theme.palette.primaryDark[400] : theme.palette.grey[300],
-        borderRadius: '10px 10px 0 0',
-        borderWidth: '1px 1px 0 1px',
-      })}
+      sx={{
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'center',
+        mt: { xs: '20px', sm: '60px' },
```

Comment:
Would the units from MUI work for margin too? Or maybe we're trying to have more specific margin in this case.

Reply:
I could use `theme.spacing` to get the values instead